web/modules/weather_data/src/Service/WeatherAlertTrait.php

```diff
@@ -67,9 +84,19 @@ public function getAlertsForLatLon($lat, $lon)
                 $output->geometry = [];
             }
 
-            $output->description = self::fixupNewlines(
-                $output->description ?? false,
+            $alertDescription = self::tryParsingDescriptionText(
+                $output->description,
             );
+            if (!is_array($alertDescription)) {
+                $output->description = self::fixupNewlines(
+                    $alertDescription ?? false,
+                );
+                $output->usesParsedDescription = false;
+            } else {
+                $output->description = $alertDescription;
+                $output->usesParsedDescription = true;
```

Comment:
There is no good way in a twig template to determine if some attribute on an object in question is or is not an array. Here the `alert.description` field will be an array with named keys. Otherwise it's just the description string itself. We use this boolean field in the twig template to determine how we display the field.